cinch/roles/jenkins_slave_container_wrapup/templates/jswarm.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
-. /etc/sysconfig/jenkins_swarm.templated
+. /var/lib/jenkins/jenkins_swarm.templated
```

Comment:
This might be better placed in **/opt** or **/usr/local/bin**.  I think that **/var/lib** should be reserved for the package manager's use.